api/model.py

```diff
@@ -37,6 +37,7 @@ def get(self):
 # set up parser for image input data
 image_parser = api.parser()
 image_parser.add_argument('image', type=FileStorage, location='files', required=True)
+image_parser.add_argument('threshold', type=float, default=0.7)
```

Comment:
Maybe we should pass in the `help` parameter here to give a brief description.
http://flask-restful.readthedocs.io/en/latest/api.html#reqparse.Argument

Reply:
that's a good idea, I'll add a help desc

Reply:
I've tested this and IIUC the help text would only be surfaced if a non float value was submitted for the threshold. Therefore it might make more sense to document it in the README rather than add a help string here. What do you think?

Reply:
@MLnick ^

Reply:
I think the README is fine then